Client/Ecosia/Extensions/HomepageViewController+Ecosia.swift

```diff
@@ -65,6 +65,14 @@ extension HomepageViewController: NTPTooltipDelegate {
     func reloadTooltip() {
         reloadView()
     }
+    
+    func ntpTooltipCloseTapped(_ tooltip: NTPTooltip?) {
+        // no-op
```

Comment:
Yes this makes sense, the no-op just indicates that this method is intentionally left blank. I'll split the delegate so it doesn't need to be declare at all, if not desired.

Reply:
Hm I went with the default implementations in a protocol extension, otherwise I would have to declare multiple delegate variables for the distance delegates as I can't constraint one variable to multiple delegates which are optional. (Only all optional or non but not every individual one).